app/sprinkles/account/composer.json

```diff
@@ -13,8 +13,7 @@
     ],
     "require": {
         "birke/rememberme" : "^2.0",
-        "nikic/php-parser" : "^1",
-        "php": ">=5.6"
```

Comment:
what...

Reply:
Nevermind. Tricked myself. PHP requirement of internal sprinkles is determined by UF at the top level.

Reply:
This should probably removed, and only used in the master composer.json (unless a specific sprinkle requires a more higher PHP version)